src/webnn_native/dml/GraphDML.cpp

```diff
@@ -743,6 +744,81 @@ namespace webnn_native { namespace dml {
         return {};
     }
 
+    MaybeError Graph::AddPad(const op::Pad* pad) {
+        auto inputsOperand = pad->Inputs();
+        DAWN_ASSERT(inputsOperand.size() == 2);
+        DAWN_ASSERT(mExpression.find(inputsOperand[0].Get()) != mExpression.end());
+        ::dml::Expression input = mExpression.at(inputsOperand[0].Get());
+        DAWN_ASSERT(mExpression.find(inputsOperand[1].Get()) != mExpression.end());
+        ::dml::Expression padding = mExpression.at(inputsOperand[1].Get());
+
+        // Workaround(mingming): If padding was added in mGraph, it must be used.
+        // This may be a dml issue: https://github.com/microsoft/DirectML/issues/133.
+        std::string name = std::to_string(time(NULL));
+        mOutputs[name] = ::dml::Identity(padding);
+
+        ::dml::TensorDimensions inputDims = input.GetOutputDesc().sizes;
+        uint32_t inputRank = inputDims.size();
+        op::Constant* paddingConstant = reinterpret_cast<op::Constant*>(inputsOperand[1].Get());
```

Comment:
Please add a member variable to decide the constant operation like https://github.com/webmachinelearning/webnn-native/blob/main/src/webnn_native/openvino/GraphIE.h#L91